### PR TITLE
Donation verification 

### PR DESCRIPTION
**Pull request by lucifer911**

### CODE REVIEW

**Comment by lucifer911** [comment]
Can someone please verify me. Thank you.

File: ONE/donations.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  recipient: 
+    name: Sunnybrook Health Sciences Centre Foundation
+    address: 2075 Bayview Avenue, KGW-01 | Toronto, Ontario M4N 3M5
+    tel: +1 416-480-4483
+    contact: foundation@sunnybrook.ca
+
+  ada: 269.02
+  usd: 495.00
+  timestamp:
```
Comment (ResponsibleStaking) [inline ONE/donations.yml:51]: timestamp is empty

File: ONE/donations.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  files:
+    - ONE/3_Sunnybrook_receipt 6/15/2021.png
+    
+    
+- ticker: ONE
+
+  recipient: 
+    name: Kids With Cancer Donation Donation
+    address:
```
Comment (ResponsibleStaking) [inline ONE/donations.yml:61]: Contact Details missing

File: ONE/donations.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  files:
+    - ONE/3_Sunnybrook_receipt 6/15/2021.png
+    
+    
+- ticker: ONE
+
+  recipient: 
+    name: Kids With Cancer Donation Donation
+    address: 
+    tel:
```
Comment (ResponsibleStaking) [inline ONE/donations.yml:62]: Tel is missing

**Review by ResponsibleStaking — CHANGES_REQUESTED** [review]

File: ONE/donations.yml (hunk 1 above)
Comment (lucifer911) [reply ONE/donations.yml]: Updated

File: ONE/donations.yml (hunk 2 above)
Comment (lucifer911) [reply ONE/donations.yml]: removed the donation receipt due to not have any contact info.

File: ONE/donations.yml (hunk 3 above)
Comment (lucifer911) [reply ONE/donations.yml]: removed the donation receipt due to not have any contact info.

**Review by lucifer911 — COMMENTED** [review]
done

**Comment by BrianLee12** [comment]
ONE/1_Sunnybrook_receipt_11-24-2020.pdf MISSING

This needs to be included in your donations.yml file for merging to continue

**Comment by lucifer911** [comment]
Hello, Just made the changes.

**Comment by nova4005** [comment]
Hey, thanks for taking a look at this today. Unfortunately just changing the name in the YAML file doesn't solve the problem, and that's why I had updated your YAML file previously. The automated checks that Github does doesn't allow spaces and it won't allow me to proceed with the merge.

That is why I was hoping you could change the actual file names and push them up. I just don't have access to do that on my end or I would.

**Comment by lucifer911** [comment]
OK, Will will do that shorty. Thank you.

**Comment by lucifer911** [comment]
Hello, I renamed the files (without spaces) and made changes in the yml file.
Thank you so much for your time.

**Review by nova4005 — APPROVED** [review]
LGTM

**Pull request merged**